plugins/org.eclipse.elk.alg.force/src/org/eclipse/elk/alg/force/graph/FNode.java

```diff
@@ -12,8 +12,6 @@
 import java.util.ArrayList;
 import java.util.List;
 
-import org.eclipse.elk.core.math.KVector;
-
```

Comment:
Could you readd the empty line?

Reply:
I can add it back, but then there are just two empty lines over the class javadoc, that the formatter will remove the next time it is applied (at least if you are using the Eclipse default too). The removal was also done by eclipse when removing the unused imports.

At the moment it is:
```
import java.util.ArrayList;
import java.util.List;

/**
 * A node in the force graph.
```
With the line added back it would be 
```
import java.util.ArrayList;
import java.util.List;


/**
 * A node in the force graph.
```

But if you prefer the latter, I can apply it.

Reply:
Ok, I think I misread the changes on my phone. In this case, no editing is needed.

Reply:
Great. Thanks. No problem, reviewing on the phone is not easy :)